### ISSUE TITLE
bug: (a11y) Angular IonSlides not working properly with screen reader enabled

### ISSUE THREAD

**Issue by VittorioEuris** [ionic-team/ionic-framework#23333]
# Bug Report

**Ionic version:**
[x] **5.x**

**Current behavior:**
Having a grid inside a slide make screen reader going to the next slide, even without a complete transition of the slide and, if i change it manually, could be red right off the screen.
Current slide cannot had any label and a11y property of SwiperJS seems not have any effects.
Pager is ignored.

**Expected behavior:**
While on a slide other slides should be hidden to the screen reader or, at least, make a complete transition.
On the other side pager should be accessibile and used to switch between pages.

**Steps to reproduce:**
Open the provided app with screen reader on and swipe right with one finger to tick between elements inside the first tab.

**Related code:**
https://github.com/VittorioEuris/ionic-a11y-issues.git

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.16.1
   Ionic Framework               : @ionic/angular 5.6.7
   @angular-devkit/build-angular : 0.1102.13
   @angular-devkit/schematics    : 11.2.13
   @angular/cli                  : 11.2.13
   @ionic/angular-toolkit        : 3.1.1

Capacitor:

   Capacitor CLI      : 3.0.0
   @capacitor/android : 3.0.0
   @capacitor/core    : 3.0.0
   @capacitor/ios     : not installed

Utility:

   cordova-res : not installed globally
   native-run  : 1.3.0

System:

   NodeJS : v14.17.0 (/home/vittorio/.nvm/versions/node/v14.17.0/bin/node)
   npm    : 6.14.13
   OS     : Linux 5.11
```

**Comment by liamdebeasi** [comment]
Thanks for the issue. Just so I understand correctly, the issue here is that screen readers are reading the wrong slide?

If so this seems similar to https://github.com/ionic-team/ionic-framework/issues/23075. There are some bugs in Swiper v5 (the technology `ion-slides` depends on) when it comes to screen readers. You could alternatively try using the Swiper library directly to get the newer version and see if it is fixed there.

**Comment by VittorioEuris** [comment]
Thanks for the report. Yes, I suppose it's similar, I also ave a problem due to the fact that in my slides there is a grid of buttons, selectable by the screen reader and the slides doesn't swipe correctly in position when scrolling through the buttons, from one slide to another, using the screen reader gestures.

I've managed to add SwiperJS, version 6.6.2, on Ionic (some problem with ion-content) but it has the same problems, I'm going to open issues to them.

**Comment by liamdebeasi** [comment]
Thanks, that makes sense. I am going to close this issue in favor of #23075, though it sounds like this may be a bug in Swiper itself. If the issue gets resolved I will post an update in #23075.
